client/state/themes/themes-ui/reducer.js

```diff
@@ -29,22 +25,9 @@ export function themesBannerVisibleReducer( state = true, { type } ) {
 	return state;
 }
 
-// Themes banner visible state
-export function themesUpworkBannerVisibleReducer( state = true, { type } ) {
-	if ( THEMES_UPWORK_BANNER_HIDE === type ) {
-		return false;
-	}
-	return state;
-}
-
 export const themesBannerVisible = withSchemaValidation(
 	themesBannerVisibleSchema,
 	themesBannerVisibleReducer
 );
 
-export const themesUpworkBannerVisible = withSchemaValidation(
```

Comment:
The `themesUpworkBannerVisible` reducer is still referenced in `client/state/themes/themes-ui/test/reducer.js`.

Reply:
Removed from tests